test/fixtures/plugin.legend/legend-doughnut-left-start-mulitiline.json

```diff
@@ -0,0 +1,24 @@
+{
+    "config": {
+        "type": "doughnut",
+        "data": {
+            "labels": ["0", "1", "2", "3", "4", "5", "6", "7", "8", "9", "10", "11", "12", "13", "14", "15", "16"],
+            "datasets": [{
+                "data": [10, 20, 30, 40, 50, 60, 70, 10, 20, 30, 40, 50, 60, 70, 10, 20, 30]
+            }]
+        },
+        "options": {
+            "responsive": false,
```

Comment:
It seems there are mixed spaces / tabs here. Also no need to specify `"responsive": false`, because its automatically handled: https://github.com/chartjs/Chart.js/blob/390a8d7b413a5ba557f8b73888bad0b19aba8ed4/test/utils.js#L57

Reply:
👍